### ISSUE TITLE
Unable to use the substituted dxgi.dll

### ISSUE THREAD

**Issue by alexv1n** [doitsujin/dxvk#1955]
I must be doing something wrong installing dxvk into my local Wine installation. I built a recent version of Wine from WineHQ git repo and built the recent version of DXVK. When launching a 3D application in this set up, I see that d3d11.dll loaded by wine is actually the replacement library from dxvk, however dxgi.dll is still from wine (the built-in dxgi.dll.so). What am I doing wrong?

**Comment by doitsujin** [comment]
You're not setting the DLL override for DXGI.

**Comment by alexv1n** [comment]
Yes, you are correct. Using `WINEDLLOVERRIDE=dxgi=n` helps. However, dxvk_setup.sh tries to do this via adding registry keys. I guess that doesn't quite work then?

**Comment by doitsujin** [comment]
I just tried this in a fresh prefix and it's working perfectly fine. Not really sure what you're having issues with.
![Bildschirmfoto](https://user-images.githubusercontent.com/25567304/109655228-543a0780-7b63-11eb-90ee-a33e1ed0fd85.png)

**Comment by alexv1n** [comment]
I think I figured out what's going on. After registry is modified in the script, it takes a few seconds for those changes to be saved into the .reg files into the prefix. And my script was packaging the prefix contents before that save operation happened.
